packages/node-cli/src/commands/outputFormatters/ManageAccountOutputFormatter.js

```diff
@@ -0,0 +1,61 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+"use strict";
+const OutputFormatter = require("./OutputFormatter");
+const ActionResultUtils = require("../../utils/ActionResultUtils");
+const NodeTranslationService = require("../../services/NodeTranslationService");
+
+const {
+   ACCOUNT_TYPE, COMMAND_MANAGE_ACCOUNT: { MESSAGES },
+} = require("../../services/TranslationKeys");
+
+const SANDBOX_ACCOUNT_ID_REGEX_PATTERN = ".+_SB\\d*$";
+const RELEASE_PREVIEW_ACCOUNT_ID_REGEX_PATTERN = ".+_RP\\d*$";
+
+class ManageAccountOutputFormatter extends OutputFormatter {
+   constructor(consoleLogger) {
+      super(consoleLogger);
+   }
+
+   formatActionResult(actionResult) {
+      if (actionResult.resultMessage) {
+         ActionResultUtils.logResultMessage(actionResult, this.consoleLogger);
+      } else if (Array.isArray(actionResult.data)) {
+         actionResult.data.forEach((message) => this.consoleLogger.result(message));
+      } else if (actionResult.data instanceof Object) {
```

Comment:
change this `else if instanceof Object` for an `else` statement

Reply:
Changed... but we could improve in the Java part to always recieve same type of Object from ManageAuth. However this would be a huge change for all tools

Reply:
We are not going to change OperationResult at the end... instead we are changing ActionResult with a new variable actionExecuted